restructing the secure ff nav bar (#6780)
harness/developer-hub: docs/feature-flags/ff-creating-flag/create-a-project.md

This topic describes how to create a project in the Harness Platform. For more information, go to [Harness Platform documentation](/docs/platform).

:::info note

> Before you create a Feature Flag, you must have [created an Organization](/docs/platform/organizations-and-projects/create-an-organization). It's also useful to know about how Access Control works in Harness, to learn more about this, go to [Manage Access Control](/docs/feature-flags/secure-ff/rbac-at-ff/manage-access-control.md).

Reviewer: ```suggestion
Before you create a Feature Flag, you need to [create an Organization](/docs/platform/organizations-and-projects/create-an-organization). It's also useful to know about how Access Control works in Harness, to learn more about this, go to [Manage Access Control](/docs/feature-flags/secure-ff/rbac-at-ff/manage-access-control.md).
```
Optional sug. I generally use present tense unless there's a good reason not to. https://developers.google.com/style/tense
